frontend/src/modules/WorkspaceSettings/pages/Users/InviteUsersForm.jsx

```diff
@@ -68,6 +69,7 @@ function InviteUsersForm({
 
   const { super_admin } = authenticationService.currentSessionValue;
   const [featureAccess, setFeatureAccess] = useState({});
+  const edition = fetchEdition();
```

Comment:
As per the modularization standards, using the edition inside components is not recommended. If there are any edition-specific changes, they should be controlled using props and base components.

@devanshu-gupta2002, please go through the other modularized components once and revert this shortly. Let’s keep it this way for now.